### ISSUE TITLE
Audio doesn't work under Cinnamon 6.0.x (Linux Mint 21.3)

### ISSUE THREAD

**Issue by AchillesBoi** [rustdesk/rustdesk#8150]
### Bug Description

When connecting remotely to Linux Mint Cinnamon, there is no audio at all.

### How to Reproduce

1) Connect to remote
2) Make sure Mute is NOT selected
3) Play any audio
4) Enjoy the silence

### Expected Behavior

Audio should be transferred through the stream. It work under GNOME, but not under Cinnamon. I was SOMEHOW able to get audio working on only one computer after installing/uninstalling 1.2.3 and then trying the Flatpak version and then moving to 1.2.4 (nightly). I tried the same steps with another Linux Mint and it didn't work. I haven't been able to make it work with a VM either.

### Operating system(s) on local side and remote side

Linux Mint 21.3 Cinnamon

### RustDesk Version(s) on local side and remote side

1.2.3 - 1.2.4

### Screenshots

No screenshot needed

**Comment by AchillesBoi** [comment]
I found the problem:

While using GNOME, RustDesk is able to know which audio source is default and choose it in every session. While using Cinnamon however, RustDesk is unable to know which audio source is default and will pick the topmost from the list of available sources.

On some of my computers (without anything plugged in), the first available audio source was not a working source and I had to manually change it using `pavucontrol`. I have to manually change it every time a new session starts.
